clients/iceberg-views/src/main/java/org/apache/iceberg/nessie/NessieExtCatalog.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (C) 2022 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.iceberg.nessie;
+
+import com.google.common.annotations.VisibleForTesting;
+import java.util.List;
+import java.util.Map;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.iceberg.BaseMetastoreCatalog;
+import org.apache.iceberg.TableOperations;
+import org.apache.iceberg.catalog.Namespace;
+import org.apache.iceberg.catalog.TableIdentifier;
+import org.apache.iceberg.view.View;
+import org.apache.iceberg.view.ViewDefinition;
+import org.apache.iceberg.view.Views;
+import org.projectnessie.error.NessieNotFoundException;
+
+/** This is a wrapper catalog that supports both Iceberg Views and Tables. */
+public class NessieExtCatalog extends BaseMetastoreCatalog implements Views {
```

Comment:
I see the need for this at the moment, but it doesn't feel right to keep this long term.
Maybe explicitly mark this as "USE AT YOUR OWN RISK / EXPERIMENTAL CODE" or so.